client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -1,17 +1,47 @@
 import { useSearchParams } from 'react-router-dom';
 
 export interface UrlQueryObject {
-  [key: string]: string | number | boolean | RangeObject;
+  [key: string]: UrlQueryValue;
```

Comment:
nice! this type was needed in the filter change, introducing it here will make that neater. 
( side note: if you update from develop, the filter methods could be updated to use `UrlQueryValue` )

Reply:
> ( side note: if you update from develop, the filter methods could be updated to use `UrlQueryValue` )

Yeah, I had to do that when updating from the feature branch to get it to compile.